ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIConstant/Constant+SearchGardenButton.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Constant+SearchGardenButton.swift
+//
+//
+//  Created by Wood on 4/19/24.
+//
+
+import Foundation
+
+extension Constant.SearchGardenButton {
+  public enum Layout {}
+  public enum ImageView {}
+}
+
+extension Constant.SearchGardenButton.Layout {
+  public static let cornerRadius: CGFloat = 12
+}
+
+extension Constant.SearchGardenButton.ImageView {
+  public static let trailingConstraint: CGFloat = -12.5
```

Comment:
```suggestion
  public static let trailingMargin: CGFloat = 12.5
```
과 같이 주고, constraint를 줄 때 앞에 -를 붙이는것도 좋아보입니다.
trailingConstraint라하면 `NSLayoutConstraint`를 연상시키는 것 같아서요

Reply:
> 과 같이 주고, constraint를 줄 때 앞에 -를 붙이는것도 좋아보입니다. trailingConstraint라하면 `NSLayoutConstraint`를 연상시키는 것 같아서요

다음 커밋에서 반영하였습니다.
https://github.com/ToDo-Garden/ToDo-Garden/pull/79/commits/63987bd7686b487a673e48522255383c73621da7